apps/web/screens/Verdicts/VerdictsList.tsx

```
@@ -148,19 +264,531 @@
+                      (value) => value !== option.value,
+                    )
+                  }
+                  return {
+                    ...previousState,
+                    [QueryParam.CASE_TYPES]:
+                      updatedCaseTypes.length === 0 ? null : updatedCaseTypes,
+                  }
+                })
+              }}
+            />
+          ))}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)